MakieCore/src/basic_plots.jl

```diff
@@ -24,7 +24,7 @@ Plots an image on range `x, y` (defaults to dimensions).
         colormap = [:black, :white],
         colorrange = automatic,
         interpolate = true,
-        fxaa = false,
+        fxaa = true,
```

Comment:
hm, I think that was on purpose false, because fxaa messes with the image sampling

Reply:
Guess that makes sense, I'll undo this